src/layers/tile/infinite.rs

```diff
@@ -82,13 +82,19 @@ fn tile_to_chunk_pos(x: i32, y: i32) -> (i32, i32) {
 }
 
 #[derive(Debug, PartialEq, Clone)]
-pub struct Chunk {
+pub(crate) struct Chunk {
     tiles: Box<[Option<LayerTileData>; Self::TILE_COUNT]>,
 }
 
 impl Chunk {
+    /// Internal infinite layer chunk width. Do not rely on this value as it might change between
+    /// versions. 
```

Comment:
With the `Chunk` struct changed to `pub(crate)`, is this warning to not rely on the value staying the same still relevant?

Reply:
Ah, shouldn't have changed the visibility here, my bad.